src/js/tracks/text-track-display.js

```diff
@@ -329,6 +347,17 @@ class TextTrackDisplay extends Component {
     // inset-inline and inset-block are not supprted on old chrome, but these are
     // only likely to be used on TV devices
     if (!this.player_.videoHeight() || !window.CSS.supports('inset-inline: 10px')) {
+      if (browser.IS_SMART_TV && !window.CSS.supports('inset', '10px')) {
+        const textTrackDisplay = this.el_;
+        const textTrackDisplayHeight = textTrackDisplay.getBoundingClientRect().height;
+
+        // textrack style updates, this styles are required to be inline
+        textTrackDisplay.style.position = 'relative';
+        textTrackDisplay.style.height = textTrackDisplayHeight + 'px';
+        textTrackDisplay.style.top = 'unset';
+        textTrackDisplay.style.bottom = '0px';
```

Comment:
```suggestion
        tryUpdateStyle(textTrackDisplay, 'position', 'relative');
        tryUpdateStyle(textTrackDisplay, 'height', textTrackDisplayHeight + 'px');
        tryUpdateStyle(textTrackDisplay, 'top', 'unset');
        tryUpdateStyle(textTrackDisplay, 'bottom', '0px');
```

this updates to use tryUpdateStyle for consistency with the rest of the code, but maybe that isn't really neccessary as the comment for tryUpdateStyle metions IE8 🤔

Reply:
add this change in the last commit, can this be re-reviewed?